### PR TITLE
Fix default multiselect height

### PR DESCRIPTION
**Pull request by skjnldsv**
To review before: https://github.com/nextcloud/nextcloud-vue/pull/1578

Switched to standard positioning to make sure the container take the content's height.
Allows design like this

Peek 26-11-2020 09-38

### CODE REVIEW

**Review by juliushaertl — APPROVED** [review]

**Comment by PVince81** [comment]
when testing this branch with the SharingInput on server, the dropdown looks broken:

image

nextcloud-vue master is fine

I guess this will probably require more adjustments for consumers.

**Comment by skjnldsv** [comment]
> when testing this branch with the SharingInput on server, the dropdown looks broken:

As mentionned: 

> To review before: #1578

Please review #1578 before :stuck_out_tongue: 
There are some fixes for this bug there

**Comment by PVince81** [comment]
> > when testing this branch with the SharingInput on server, the dropdown looks broken:
> 
> As mentionned:
> 
> > To review before: #1578
> 
> Please review #1578 before stuck_out_tongue
> There are some fixes for this bug there

ah I was confused and I thought "To review before X" meant that this PR here needs to be reviewed before X, not the other way around. I usually add an entry "- [ ] REQUIRES: X" so people know there's another dependency to look at first.

**Comment by skjnldsv** [comment]
> ah I was confused and I thought "To review before X"

Right, englisch is hart! :see_no_evil: 
Now I see how it can be read both ways :stuck_out_tongue_closed_eyes:, sorry for that! :hugs:

**Comment by PVince81** [comment]
#1578 is merged, rebase ?

**Comment by skjnldsv** [comment]
Peek 26-11-2020 09-38
Rebased and fixed Multiselect inner height (with docs)

File: src/components/ListItemIcon/ListItemIcon.vue [hunk 1]
```diff
@@ -189,8 +198,11 @@ export default {
 		},
 
 		cssVars() {
+			// Don't use margin to calculate the height if noMargin
+			const margin = this.noMargin ? 0 : this.margin
```
Comment (PVince81) [inline src/components/ListItemIcon/ListItemIcon.vue:202]: why not take "margin='0'" as property for disabling ? probably we don't want people supplying custom margins I guess ?

File: src/components/Multiselect/Multiselect.vue [hunk 2]
```diff
@@ -110,33 +111,34 @@ By specifying `:user-select="true"`, you can benefit from a fully formatted layo
...
+		</template>
+	</Multiselect>
 </template>
 
 <script>
 import Multiselect from '../index'
+
+// Building fake data for the docs
+const options = ['admin', 'user1', 'user2', 'guest', 'group1']
+const formatedOptions = options.map(item => {
```
Comment (PVince81) [inline src/components/Multiselect/Multiselect.vue:126]: minor typo => "formattedOptions"

**Review by PVince81 — APPROVED** [review]
:+1: see minor comments

File: src/components/ListItemIcon/ListItemIcon.vue (hunk 1 above)
Comment (skjnldsv) [reply src/components/ListItemIcon/ListItemIcon.vue:202]: > probably we don't want people supplying custom margins I guess ?

Yep :/

**Pull request merged**